src/Sulu/Bundle/AdminBundle/Resources/js/components/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import React from 'react';
+import type {ElementRef} from 'react';
+import classnames from 'classnames';
+import Icon from '../Icon';
+import checkboxStyles from './checkbox.scss';
+
+type Props = {
+    checked: boolean,
```

Comment:
Do you think we really need that? Couldn't find a checkbox property in husky, but the [toggler in husky](https://github.com/massiveart/husky/blob/a2d909f144906ca1bb412b387135bc7d87c36b32/husky_components/toggler/main.js) (which is basically a pimped checkbox) doesn't have a value property either.